docs/reference/API/element.md

```diff
@@ -0,0 +1,295 @@
+# `Element`
+
+The `Element` API is a helpful way to create and manipulate elements in the DOM. It is a wrapper around the native DOM API, and is designed to be as intuitive as possible.
+
+## Methods and Properties
+
+| Property | Description                             |
+|----------|-----------------------------------------|
+| `element`   | Returns the element with the given ID. |
+| `id`        | Returns the element's ID.              |
+| `value`    | Returns the element's value.           |
+| `innerHtml` | Returns the element's inner HTML.      |
+
+
+
+| Method               | Description                            |
+|----------------------|--------------------------------------------------------------|
+| `write`                | Writes `value` to element and handles various mime types. `append` defaults to `False`, if set to true, it will create a child element.          |
+| `clear`                | Clears the element's value or content. |
+| `select`               | Select element from `query`.    |
```

Comment:
We can be more specific about `query` coming from `querySelector` we can link for MDN here.